iot/api-client/manager/pom.xml

```diff
@@ -36,7 +36,19 @@
     <maven.compiler.target>1.7</maven.compiler.target>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>Eclipse Paho Repo</id>
+      <url>https://repo.eclipse.org/content/repositories/paho-releases/</url>
```

Comment:
don't think this is required as it's in [Maven Central](http://search.maven.org/#search%7Cga%7C1%7Corg.eclipse.paho.client.mqttv3)

Reply:
Done.